crypto/keys/public_keys.go

```diff
@@ -25,9 +25,19 @@ import (
 	"fmt"
 	"io/ioutil"
 
+	"github.com/google/trillian/crypto/keyspb"
 	"github.com/google/trillian/crypto/sigpb"
 )
 
+// ToPublicKeyPB returns a keyspb.PublicKey from crypto.PublicKey.
+func ToPublicKeyPB(pubKey crypto.PublicKey) (*keyspb.PublicKey, error) {
```

Comment:
Suggestion: Write a simple test for ToPublicKeyPB, since you plan to depend directly on it?